BlazorLeaflet/BlazorLeaflet/Utils/ICanUpdateStyleLayer.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace BlazorLeaflet.Utils
+{
+    /// <summary>
+    /// Interface to make sure SetStyle is only called on leaflet layers actually supporting it.
+    /// https://leafletjs.com/reference-1.6.0.html
+    /// </summary>
+    public interface ICanUpdateStyleLayer
```

Comment:
Let's rename it into **IStyeable** for the sake of keeping names shorter and as detailed.